test.ps1

```diff
@@ -0,0 +1,40 @@
+param(
+  [switch] $prerelease
+)
+
+choco uninstall clang --force -y
+if ($prerelease) {
+  choco install clang --verbose --debug --prerelease --force -y -s ".;https://chocolatey.org/api/v2/"
```

Comment:
The --sources param takes a CSV list of sources to choose from. Here, we don't want to take a remote source, so I'll remove that.